narwhal/Docker/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# fast fail.
+set -e
+
+DIR="$( cd "$( dirname "$0" )" && pwd )"
```

Comment:
Nit: `DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"` is [more robust](https://stackoverflow.com/questions/35006457/choosing-between-0-and-bash-source).

Reply:
oh nice, thanks for the tip!